### PR TITLE
T gigo/unit testing

### LINKED ISSUE #12: Add unit test for PurgeTags and PurgeManifests

### PR DESCRIPTION
**Pull request by ggonzalere**
This PR adds the unit testing for the purge command
The test is in the file purge_test.go, some changes were required for the mock test but there is no functionality change.

- 

Fixes #12

### CODE REVIEW

**Comment by northtyphoon** [comment]
> // Licensed under the MIT License.

Is the file generated from the ACR data plane open api spec (swagger)?

---
Refers to: cmd/api/acrsdk.go:2 in 3324de7. [](commit_id = 3324de79f73e4918e3f425bc456a3699985845b0, deletion_comment = False)

File: cmd/acr/purge.go [hunk 1]
```diff
@@ -209,7 +210,7 @@ func ParseDuration(ago string) (time.Duration, error) {
 // returns a pointer to a slice that contains the tags that will be deleted, the last tag obtained through the AcrListTags method
 // and an error in case it occurred, the fourth return value contains a map that is used to determine how many tags a manifest has
 func GetTagsToDelete(ctx context.Context,
```
Comment (northtyphoon) [inline cmd/acr/purge.go]: >GetTagsToDelete [](start = 5, length = 15)

Do you need to export the functions in the module? If not, should you just rename them, eg, getTagsToDelete

**Comment by ggonzalere** [comment]
> acrsdk

The acrsdk.go file is a wrapper to the autorest generated files, for now they are in the acr folder (the client, the models and the version files), but in the future they can be referenced from the azure-sdk-for-go repo. @northtyphoon

File: cmd/acr/purge.go (hunk 1 above)
Comment (ggonzalere) [reply cmd/acr/purge.go]: All right, I've changed them all in the purge.go file, in the other ones I think they were unexported.

**Pull request merged**